mavros/src/plugins/global_position.cpp

```diff
@@ -116,8 +116,8 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 	ros::Subscriber gp_set_global_origin_sub;
 	ros::Subscriber hp_sub;
 
-	std::string frame_id;       //!< origin frame for topic headers
-	std::string child_frame_id; //!< body-fixed frame for topic headers
+	std::string frame_id;				//!< origin frame for topic headers
```

Comment:
Two less tabs and it will be good :)

Reply:
What is the convention for Tab - 8 spaces?

Reply:
It's the current one being used. Also accepted by the `uncrustify` config.

Reply:
Done.